packages/libs/react-ui/src/components/Card/Card.css.ts

```diff
@@ -25,15 +29,24 @@ export const fullWidthClass = style({
 export const stackClass = style([
   sprinkles({ marginY: 0 }),
   {
+    margin: 0,
```

Comment:
can this be moved to sprinkles?

Reply:
it was actually redundant, removed it for simplicity on the last commit